debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleValueConverters.java

```diff
@@ -149,11 +169,23 @@ public ValueConverter converter(Column column, Field fieldDefn) {
         return super.converter(column, fieldDefn);
     }
 
+    private Object getFloatConverter(Column column, Field fieldDefn, Object data) {
+        if (data instanceof BigDecimal) {
+            return ((BigDecimal) data).floatValue();
+        }
+        return convertVariableScale(column, fieldDefn, data);
+    }
+
     private ValueConverter getNumericConverter(Column column, Field fieldDefn) {
         if (column.scale().isPresent()) {
             Integer scale = column.scale().get();
 
             if (scale <= 0) {
+                // Boolean represtented as Number(1,0)
```

Comment:
See above, it seems we couldn't export NUMBER(1, 0) as number any longer with that?

Reply:
Reverted this change.